### DISCUSSION TITLE
Na 5 Dagen gelukt !

### DISCUSSION THREAD

**Discussion by dadalouis** [thieu-b55/Slimme-Digitale-Energie-Meter#6] [Show and tell]
https://github.com/thieu-b55/Slimme-Digitale-Energie-Meter
dit is de handleiding die ik heb gebruik, met wat kleine aanpassingen is het gelukt om de meter uit te lezen via web .222
ipv een bc547 heb ik een 2N3904  gebruiken en 1x een extra 1k ohm weerstand er tussen.
t eerst wat ik heb gemaakt ;p

nu wil deze noob graag via mqtt in home assistant de boel uit lezen. is daar een voorbeeld van?
[IMG_5298.HEIC.pdf](https://github.com/thieu-b55/Slimme-Digitale-Energie-Meter/files/13385132/IMG_5298.HEIC.pdf)
[IMG_5299.HEIC.pdf](https://github.com/thieu-b55/Slimme-Digitale-Energie-Meter/files/13385133/IMG_5299.HEIC.pdf)

**Comment by thieu-b55** [comment]
dadalouis,

Ik vrees dat ik je daarover geen antwoord kan geven.
Met mqtt en home assistant heb ik geen ervaring.

vriendelijke groeten en succes met de hobby,
thieu

Op vr 17 nov 2023 om 00:39 schreef dadalouis ***@***.***>:
